eZ/Publish/Core/Persistence/Legacy/Content/Type/Gateway/DoctrineDatabase.php

```diff
@@ -1323,4 +1323,70 @@ public function getSearchableFieldMapData()
 
         return $statement->fetchAll(\PDO::FETCH_ASSOC);
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function updateContentTypeCopyIdentifier($typeId, $baseIdentifier)
```

Comment:
> (...) `copy_of_<identifier>_$typeId` (...)

@andrerom: what about the case when someone decides to copy once again the same type? This would create a duplicate.